arcade/sprite.py

```diff
@@ -39,6 +39,15 @@
 FACE_DOWN = 4
 
 
+def points_by_dimensions(width, height):
```

Comment:
If we add a function like this:
* Definitely needs type hinting added
* If used internally, it should start with an underscore
* If intended for universal use, should go into geometery.py I think? And have documentation added. Might be able to come up with a more explanitory function name.

, it probably needs to go into

Reply:
Yes, I hadn't intended on getting this code merged, I was mostly looking for feedback on the issue, as I'm not sure this solution is the desired one.